src/components/curing-pages/Sensors-list-page/Sensors-list/list.jsx

```diff
@@ -29,7 +29,7 @@ function List({ Curing = true }) {
   React.useEffect(() => {
     fetch(`${endpointUrl}/get-sensors`)
```

Comment:
We need `credentials: "include"` here as well - the BE needs to check if jwt cookie is authorised before returning data.